### ISSUE TITLE
Installing software on windows

### ISSUE THREAD

**Issue by LinguList** [digling/calc-seminar#1]
We need to know how to install python3, git, etc. on Windows!

**Comment by malungvarg** [comment]
im still trying to install cygwin and the suggested packages for Windows 10, but im unsure about how to find / select the packages. they seem to be hidden in different subdirectories. when I run the installer and search for e.g. "git", several related files are found, but which of these should I install???
![image](https://user-images.githubusercontent.com/22367435/39059900-161dcb6c-44c0-11e8-9f15-f5e35d903dc5.png)
![image](https://user-images.githubusercontent.com/22367435/39059931-2dea3578-44c0-11e8-90e1-04bbe0d8007e.png)

**Comment by tresoldi** [comment]
Yes, cygwin can be confusing in this sense. In the specific case of `git`, you need the one named just that (5984k).

You usually don't need anything with debug or devel. Luckily for the seminar, once you have python and git sorted out, you shouldn't need to deal with any arcane cygwin configuration. :)

Be sure to keep asking for help if you need!
